### ISSUE TITLE
Can't get the script to work

### ISSUE THREAD

**Issue by trasdahl** [ulken/playmo#4]
Hi,

I have installed the extension and turned it on (hopefully), but none of the keyboard shortcuts are doing anything. Any tips on debugging? I am new to Chrome extensions :)

**Comment by ulken** [comment]
Hi @trasdahl!

Glad to see you tried it out.

I am pretty certain you haven’t done anything wrong. I managed to break it trying to fix a perk in one of the services. I haven’t actually used any of them enough lately to get around to fix it.

I’ll try to tend to it this week and provide installation instructions.

If it still doesn’t work I can give you some hints around debugging.

I’ll get back to you!

PS. Which service are you using?

**Comment by trasdahl** [comment]
Thanks.

I mainly use HBO Nordic.

**Comment by ulken** [comment]
OK, so I have figured out that the extension script doesn't load correctly on HBO when navigating from the main site. Probably due to HBO doing client side routing which doesn't trigger full page loads. It _should_ however work if you are watching a video already and **reload** the page.

The reason it works like that now is I tried to be smart and only load the script where applicable, but guess I'll have to resort to matching the entire domain.
